test/inductor/test_fp8.py

```diff
@@ -450,13 +452,18 @@ def linear(x_fp8, x_inverse_scale, w_t_fp8, w_inverse_scale, bias):
             linear_compiled = torch.compile(
                 linear, backend="inductor", mode="max-autotune"
             )
-            y_compiled = linear_compiled(
+            y_compiled, code = run_and_get_code(
+                linear_compiled,
                 x_fp8,
                 x_inverse_scale,
                 w_t_fp8,
                 w_inverse_scale,
                 bias,
             )
+
+            src = code[0]
+            if "triton_tem_fused__scaled_mm" in src:
```

Comment:
this is a bit fragile, if naming convention changes this test won't check anything